cms/static/js/views/settings/grading.js

```diff
@@ -161,8 +173,9 @@ var GradingView = ValidatingView.extend({
             var newBar = this.gradeCutoffTemplate({
                 descriptor : cutoff['designation'] ,
                 width : nextWidth,
+                contenteditable: true,
                 removable : removable });
-            gradelist.append(newBar);
+            HtmlUtils.append(gradelist,  newBar);
```

Comment:
This would be more clear without using `newBar`, and following the format used below (see `gradeCutoffTemplate`) for calling the template in the `append()` function.